plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -457,12 +468,14 @@ static void close_video(struct ffmpeg_data *data)
 
 static void close_audio(struct ffmpeg_data *data)
 {
-	for (size_t i = 0; i < MAX_AV_PLANES; i++)
-		circlebuf_free(&data->excess_frames[i]);
+	for (int idx = 0; idx < data->num_audio_streams; idx++) {
+		for (size_t i = 0; i < MAX_AV_PLANES; i++)
+			circlebuf_free(&data->excess_frames[idx][i]);
+		av_freep(&data->samples[idx][0]);
```

Comment:
Consider adding a blank line above this line to give some visual separation from the preceding `for` loop.